tutorials/README.rst

```diff
@@ -4,10 +4,27 @@ DC2 Tutorials
 This directory contains tutorial and demonstration notebooks convering how to access and use the DC2 datasets.
 See the index table below for links to the notebook code, and an auto-rendered view of the notebook with outputs.
 
-If you are unsure on exactly how to start to use these tutorials, you may want to look at 
-`this step-by-step guide on Confluence <https://confluence.slac.stanford.edu/x/Xgg4Dg>`_ *(DESC members only)*.
+Notes for Tutorial Users
+------------------------
+
+*(Notes for tutorial contributors are at the bottom of the page.)*
+
+* If you are unsure on exactly how to start to use these tutorials, you may want to look at
+  `this step-by-step guide on Confluence <https://confluence.slac.stanford.edu/x/Xgg4Dg>`_ *(DESC members only)*.
+
+* If you want to obtain an overview of the different types of DC2 data products and their access methods,
+  visit `this DC2 Data Product Overview page on Confluence <https://confluence.slac.stanford.edu/x/oJgHDg>`_ *(DESC members only)*.
```

Comment:
Should we change the `*(DESC members only)*` to something like `*(only accessible to DESC members)*` so it doesn't sound like we are opposed to non-DESC members participating (are we?), but that we are limited by the system we have set up?

It acknowledge I am splitting hairs.